scripts/enrich_rulings_ai.py

```
@@ -12,96 +14,146 @@
-
-    combined_codes = set(existing_related_codes)
-    if simulated_new_code != source_card_code:
-        combined_codes.add(simulated_new_code)
-
-    # Remove source_card_code if it accidentally got added
-    combined_codes.discard(source_card_code)
-
-    return sorted(combined_codes)
+    prompt = f"""
+    Given the following ruling text for the card with code '{source_card_code}', identify any other card codes mentioned in the text.
+    Card codes are 5-digit strings.
```

Ruff (0.11.9)

Line too long (133 > 100)

(E501)